src/pages/admin-ui-sdk/reference/menu-app.md

```diff
@@ -1,6 +1,62 @@
 ---
 title: MenuApp component
-description: 
+description: MenuApp component props and usage.
 ---
 
 # MenuApp component
+
+`MenuApp` adds a menu item to the Commerce Admin UI menu page. This component selects the DOM element on the Admin menu page that matches its `rootElmSelector` prop. This is where the `MenuApp` renders a new menu item with a URL to the out-of-process App Builder page used to add Commerce features.
```

Comment:
"Commerce Admin UI menu page" (first sentence) seems off. I don't think you need "UI" at all. Maybe change to "Commerce Admin navigation".

You subsequently use the term several times, but see if you can write around that. We've got components that create menus and pages, so I'd prefer to avoid the compound noun that uses both words. Or maybe there's nothing that can be done.